app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Idv
+  module HybridMobile
+    module Socure
+      class DocumentCaptureController < ApplicationController
+        include Idv::AvailabilityConcern
+        include DocumentCaptureConcern
+        include Idv::HybridMobile::HybridMobileConcern
+
+        before_action :check_valid_document_capture_session
+
+        def show
+          Funnel::DocAuth::RegisterStep.new(document_capture_user.id, sp_session[:issuer]).
+            call('hybrid_mobile_socure_document_capture', :view, true)
+
+          # document request
+          document_request = DocAuth::Socure::Requests::DocumentRequest.new(
+            document_capture_session_uuid: document_capture_session_uuid,
+            redirect_url: idv_hybrid_mobile_socure_document_capture_url,
```

Comment:
i think you can only send a redirect_url that exists or the document request will fail. i think you'd either have to remove the redirect_url from this request or re-add the update action and return a status (ie: 200)

Reply:
Removed.

Reply:
chatted w/ @jmax-gsa and will add back dummy #update actions in socure document capture controllers for a path forward 👍🏿